osi_lane.proto

```diff
@@ -54,56 +54,58 @@ message Lane
 
         // The lane's center line (as a list of segments).
         //
-		// The center line describes the middle of the lane. 
-		//
+        // The center line describes the middle of the lane.
+        //
         // \image html highway_exit.png "Center lines as yellow dotted lines."
         //
         // \attention The points describing the center line might be set at
-        // arbitrary distances. When the points are pairwise linearly connected, the
-        // lateral distance to the real ideal line (as used by the simulation
-        // environment internally) must not exceed 5cm. As shown in the following
-        // image:
-        //
-        // \image html line_approximation_error.png "Approximation error as green line."
-        //
-		// \note The center line is the line that a typical vehicle follows more or 
-		// less (depending on the situation, a little more to the left or right of the 
-		// center line). Thus, if the road narrows (e.g. at a construction site) this 
-		// line is supposed to describe the trajectory that the vehicle should follow. 
-        // Depending on the size of the vehicle, the vehicle may not be able to 
-        // orient itself directly on the center line due to its dimensions, but 
-        // may have to deviate from it in order to avoid e.g. a collision.
-        // The intended direction of travel on the lane is given by the
-        // direction defined by the sequence of points forming the center line.
+        // arbitrary distances. When the points are pairwise linearly connected,
+        // the lateral distance to the real ideal line (as used by the
+        // simulation environment internally) must not exceed 5cm. As shown in
+        // the following image:
+        //
+        // \image html line_approximation_error.png "Approximation error as
+        // green line."
+        //
+        // \note The center line is the line that a typical vehicle follows more
```

Comment:
I would change this paragraph as follows:

The center line is the line that a typical vehicle follows more or less (depending on the situation, a little more to the left or right of the center line). The intended direction of travel on the lane is given by the direction defined by the sequence of points forming the center line.